tests/api.rs

```diff
@@ -86,7 +86,9 @@ async fn test_tenants_and_users() {
         .unwrap();
 
     // Verify tenant properties.
-    let tenants = client.list_tenants().await.unwrap();
+    // sort tenants by name to ensure ordering. Default ordering is by tenantId.
+    let mut tenants = client.list_tenants().await.unwrap();
+    tenants.sort_by(|a, b| a.name.cmp(&b.name));
```

Comment:
I think you should be able to use the simpler `sort_by_key` method here!

```suggestion
    tenants.sort_by_key(|tenant| &tenant.name);
```